packages/sdk/src/api/entities/VCLDeepLink.ts

```diff
@@ -1,23 +1,24 @@
 export default class VCLDeepLink {
     constructor(public value: string) {}
 
-    get issuer(): Nullish<string> {
-        return this.generateUri(VCLDeepLink.KeyIssuer, true);
-    }
-
     get requestUri(): Nullish<string> {
         return this.generateUri(VCLDeepLink.KeyRequestUri);
     }
 
+    // get did(): Nullish<string> {
+    //     return (
+    //         this.requestUri?.getUrlSubPath(VCLDeepLink.KeyDidPrefix) ??
+    //         this.issuer?.getUrlSubPath(VCLDeepLink.KeyDidPrefix)
+    //     );
+    // }
     get did(): Nullish<string> {
         return (
-            this.requestUri?.getUrlSubPath(VCLDeepLink.KeyDidPrefix) ??
-            this.issuer?.getUrlSubPath(VCLDeepLink.KeyDidPrefix)
+            this.retrieveQueryParam(VCLDeepLink.KeyIssuerDid) ?? this.retrieveQueryParam(VCLDeepLink.KeyInspectorDid)
```

Comment:
If the host of the URL is `issue` then use `VCLDeepLink.KeyIssuerDid` but if its `inspect` then use `VCLDeepLink.KeyInspectorDid`. Finally in both cases `this.requestUri?.getUrlSubPath(VCLDeepLink.KeyDidPrefix)` should still be there as a fallback for old agents that are still running